src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/SDKVersionTelemetryServiceComponent.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Telemetry;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies
+{
+    /// <summary>
+    /// For maintaining light state about the SDK version used in a project
+    /// </summary>
+    [Export(ExportContractNames.Scopes.ConfiguredProject, typeof(IProjectDynamicLoadComponent))]
```

Comment:
You are reporting this for every single configuration that ever gets loaded, but only ever report the "Active" config. That does not seem correct.

Reply:
We now report for each configuration that is loaded.  Let me know if you think there is a better approach here.

Reply:
Okay, I understand. My comment before was that you needed to be consistent with property reading:

- Use the active configuration's project properties & export into UnconfiguredProject scope

-or-

- Use each configuration's project properties & export into ConfiguredProject scope

I suspect we really just want the first, you were mixing both in previous attempts.